src/mbgl/gl/attribute.hpp

```diff
@@ -24,6 +24,7 @@ using NamedAttributeLocations = std::vector<std::pair<const std::string, Attribu
 class Context;
 void bindAttributeLocation(Context&, ProgramID, AttributeLocation, const char * name);
 std::set<std::string> getActiveAttributes(ProgramID);
```

Comment:
Is it used anywhere else?

Reply:
Implementation is removed, should be removed as well?

Reply:
Done. Thanks. `class Context` and unnecessary includes removed, too with [patch #2](https://github.com/mapbox/mapbox-gl-native/pull/14482/commits/c1e12c80d57c6e56438a23df533db969c893df77).